Gemfile

```diff
@@ -80,3 +83,6 @@ gem 'capistrano-rails-console'
 
 # Production logging
 gem 'rails_12factor', group: :production
+
+# Environment Variables
+gem 'dotenv-rails', groups: [:development, :test]
```

Comment:
We have a `:development, :test` group [here](https://github.com/cesium/intra.seium.org/blob/6133e24b4cddf2055217a1d884725045cc6de3c7/Gemfile#L58). You can just put it together. Also, while you're at it, please fix that indentation.